src/hpccplatform/launchConfig.ts

```diff
@@ -222,19 +227,17 @@ export class LaunchConfig implements LaunchRequestArguments {
     }
 
     get user() {
-        const creds = credentials[this.serverAddress];
-        if (creds?.verified) {
-            return creds.user;
-        }
+        // Always return from config - secure storage is checked in getStoredCredentials
         return config(this._lcID, "user", "vscode_user");
     }
 
     get password() {
-        const creds = credentials[this.serverAddress];
-        if (creds?.verified) {
-            return creds.password;
-        }
-        return config(this._lcID, "password", "");
+        // Password should not be retrieved from config - use getStoredCredentials instead
+        return "";
+    }
```

Comment:
Remove if not used, otherwise fetch the password using the getStoredCredentials method